ua/org.eclipse.tips.tests/src/org/eclipse/tips/core/TipImageBas64Test.java

```diff
@@ -41,13 +41,12 @@ public void testAssertWidth() {
 
 	@Test
 	public void testSetExtension() {
-		// assertTrue(getTipImage().getIMGAttributes(19, 10).contains("png"));
+		assertTrue(getTipImage().setExtension("png").getBase64Image().contains("png"));
```

Comment:
Testing the default.
```suggestion
		assertTrue(getTipImage().getBase64Image().contains("png"));
```